tests/Hedgehog.Tests/GenTests.fs

```diff
@@ -5,23 +5,35 @@ open Hedgehog.Gen.Operators
 open TestDsl
 
 let genTests = testList "Gen tests" [
-    yield! testCases "dateTime creates System.DateTime instances" [ 8; 16; 32; 64; 128; 256; 512 ] <| fun count->
-        let actual = Gen.dateTime (Range.constant System.DateTime.MinValue System.DateTime.MaxValue) |> Gen.sample 0 count
+    yield! testCases "dateTime creates System.DateTime instances"
+        [ 8; 16; 32; 64; 128; 256; 512 ] <| fun count->
```

Comment:
I'd like to see if we can get rid of the `<|` in these tests, only because we decided to do it in #276.

Reply:
Yes, I think so too. Let's do this on a separate PR.